packages/code-connect/components/ClipboardCopy/ClipboardCopyBasic.figma.tsx

```diff
@@ -0,0 +1,26 @@
+import figma from '@figma/code-connect';
+import { ClipboardCopy } from '@patternfly/react-core';
+
+/**
+ * PatternFly ClipboardCopyInlineCompact component integration for Figma Code Connect
+ */
+
+figma.connect(
+  ClipboardCopy,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=9914-75835&m=dev',
+  {
+    props: {
+      // enum
+      isActive: figma.enum('State', { Active: true }),
+      isReadOnly: figma.enum('State', { 'Read only': true }),
+      isExpanded: figma.enum('State', { Expanded: true }),
```

Comment:
is there a way in figma to make an expandable clipboardCopy variant? If so, then `isExpanded` can be passed, but also the `variant` needs to be `'expansion'`

Reply:
We can create an expansion var that references `figma.enum('State', { Expanded... }`

`false: undefined` should be included to return "undefined" if `state === !expanded`, otherwise Figma will complain

```
      expansion: figma.enum('State', { 
        Expanded: 'expansion',
        false: undefined
      }),
```